proposals/container-build-plan-spec.md

```diff
@@ -39,7 +39,30 @@ Although looking similar, the structure and semantics of similarly named propert
 
   "layers": [
     {
-      "type": "fileEntries"
+      "type": "platformDependent",
+      "entries": {
+        "amd64_linux": {
```

Comment:
I don't have great thoughts on how to represent the hybrid key `Platform` as a JSON key, so went with `<arch>_<os>`. 🤷‍♂️

Reply:
a common format used in `Dockerfile` and other tools is `os/arch[/variant] (e.g. linux/amd64)`